.gitignore

```
@@ -1,5 +1,17 @@
 .idea
+.DS_Store
 
 # Ruby gem
 ruby/*.gem
 ruby/.rspec_status
+
+# Python
+
+python/bin
+python/lib
+python/dist
+python/.Python
+python/include
+python/build
+python/*.egg-info
+python/*/__pycache__
```

```suggestion
python/*/__pycache__

```
Newline eof (I think there is a merge conflict in this file fyi)

---

Should i rebase the master branch?

---

Fixed the conflict 👍